src/main/resources/database/changes/release-11/changelog.yml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+
+  - changeSet:
+      id: 11.-1
```

Comment:
sorry to be fussy, but I'm really not a fan of -1 version id. Can we just make it 11.1 pls?